lib/Db/Synchronization.php

```diff
@@ -103,17 +103,17 @@ public function jsonSerialize(): array
 			'sourceHash' => $this->sourceHash,
 			'sourceTargetMapping' => $this->sourceTargetMapping,
 			'sourceConfig' => $this->sourceConfig,
-			'sourceLastChanged' => $this->sourceLastChanged,
-			'sourceLastChecked' => $this->sourceLastChecked,
-			'sourceLastSynced' => $this->sourceLastSynced,
+			'sourceLastChanged' => isset($this->sourceLastChanged) ? $this->sourceLastChanged->format('c') : null,
+			'sourceLastChecked' => isset($this->sourceLastChecked) ? $this->sourceLastChecked->format('c') : null,
+			'sourceLastSynced' => isset($this->sourceLastSynced) ? $this->sourceLastSynced->format('c') : null,
```

Comment:
Dit is ok, maar wel 2 opmerkingen / good 2 know:
hier zouden we misschien isset() === true willen voor duidelijkheid (codacy)
en one line if statements zou codacy afkeuren / willen we vermeiden

Reply:
tIs meer een switch ipv if statement. maar heb hem nu aangepast